db/db_versioned.go

```diff
@@ -110,11 +135,17 @@ func (b *BoltDBVersioned) Get(version uint64, ns string, key []byte) ([]byte, er
 	if !b.db.IsReady() {
 		return nil, ErrDBNotStarted
 	}
+	if _, ok := b.vns[ns]; !ok {
+		return b.db.Get(ns, key)
+	}
 	// check key's metadata
 	if err := b.checkNamespaceAndKey(ns, key); err != nil {
 		return nil, err
 	}
 	_, v, err := b.get(version, ns, key)
+	if err == ErrDeleted {
```

Comment:
using `errors.Is` or `errors.cause` is more classic way in golang, avoiding potential bugs in future maintenance

Reply:
updated